### ISSUE TITLE
Module rtt_stdio and vfs not working together

### ISSUE THREAD

**Issue by basilfx** [RIOT-OS/RIOT#9743]
#### Description
I'm unable to run some examples with `rtt_stdio` in combination with `vfs`. Stdin seems to be looping, and no input is accepted.

I can confirm it somewhat works, as I can see the RIOT boot message.

I am using an EFM32 SLSTK3402a, but I don't think it's a board issue.

#### Steps to reproduce the issue
* Checkout master
* Modify `examples/default` and add `USEMODULE += rtt_stdio vfs`
* Compile and flash

Depending on your board, start a RTT terminal. I use JLink in combination with JLinkRTTClient. This is different from the included method, which doesn't work either (but doesn't print the characters).

* Now remove `vfs` from `USEMODULES`
* Compile and flash again.

This time, a usable terminal is presented.

#### Expected results
A usable shell is presented

#### Actual results
A lot of this (keeps repeating):

```
 > > > > > > > > > > > > > > > > > > > > > > > > >> > > > > > > > > > >  > > > > > > > > > > > > > > > > > > > > > > > > >> > > > > > > > > > >  > > > > > > > > > > > > > > > > > > > > > > > > >> > > > > > >
 > > > >  > > > > > > > > > > > > > > > > > > > > > > > > >> > > > > > > > > > >  > > > > > > > > > > > > > > > > > > > > > > > > >> > > > > > > > > > >  > > > > > > > > > > > > > > > > > > > > > > > > >> > 
> > > > > > > > >  > > > > > > > > > > > > > > > > > > > > > > > > >> > > > > > > > > > >  > > > > > > > > > > > > > > > > > > > > > > > > >> > > > > > > > > > > 
```

#### Versions
```
Installed compiler toolchains
-----------------------------
             native gcc: Apple LLVM version 10.0.0 (clang-1000.10.40.1)
      arm-none-eabi-gcc: arm-none-eabi-gcc (GNU Tools for Arm Embedded Processors 7-2018-q2-update) 7.3.1 20180622 (release) [ARM/embedded-7-branch revision 261907]
                avr-gcc: missing
       mips-mti-elf-gcc: missing
             msp430-gcc: missing
   riscv-none-embed-gcc: missing
                  clang: Apple LLVM version 10.0.0 (clang-1000.10.40.1)

Installed compiler libs
-----------------------
   arm-none-eabi-newlib: "3.0.0"
    mips-mti-elf-newlib: missing
riscv-none-embed-newlib: missing
               avr-libc: missing (missing)

Installed development tools
---------------------------
                  cmake: cmake version 3.12.0
               cppcheck: missing
                doxygen: missing
                 flake8: missing
                    git: git version 2.18.0
             coccinelle: missing
```

**Comment by basilfx** [comment]
Maybe @samkumar has an idea?

**Comment by basilfx** [comment]
Issue can be closed if #9790 gets merged.

**Comment by basilfx** [comment]
This can be closed now that #9790 is merged.
